ISISComputingGroup/ibex_gui
# Ticket 1024 - Fix unit test configurations

https://trac.isis.rl.ac.uk/ICP/ticket/1024

To test this:
*Check everything is using 1.7 (in Eclipse look at the version next to JRE System Library for all the projects)
*Check log.tests behaves correctly and has the org.junit import
*Check experimentdetails.tests runs in the Tycho build
*Check that moved tests in dae.tests and synoptic.tests are now being run successfully by Tycho
*Check that synoptic.tests runs correctly in Eclipse

After merging take a look at Jenkins and see if all the tests mentioned above now run.
